cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
I might be misunderstanding it. Isn't `self.arch.name` passed to `get_relocations()`? How can you access the `arch` argument here in this file by `self.arch`?

If this is just descriptive, you probably want to change them to comment strings.